mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -519,15 +519,31 @@ func TestAccContainerNodePool_withUpgradeSettings(t *testing.T) {
 		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
 		Steps: []resource.TestStep{
 			{
-				Config: testAccContainerNodePool_withUpgradeSettings(cluster, np, 2, 3),
+				Config: testAccContainerNodePool_withUpgradeSettings(cluster, np, 2, 3, "SURGE", "", 0, 0.0, ""),
 			},
 			{
 				ResourceName:      "google_container_node_pool.with_upgrade_settings",
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
 			{
-				Config: testAccContainerNodePool_withUpgradeSettings(cluster, np, 1, 1),
+				Config: testAccContainerNodePool_withUpgradeSettings(cluster, np, 1, 1, "SURGE", "", 0, 0.0, ""),
+			},
+			{
```

Comment:
Running some tests locally. When I only run the last two steps of the test (tests with only BLUE_GREEN config). I get errors like:
```
provider_test.go:315: Step 1/4 error: Error running apply: exit status 1
        
        Error: Surge upgrade settings may not be changed when surge strategy is not enabled
        
          with google_container_node_pool.with_upgrade_settings,
          on terraform_plugin_test.tf line 13, in resource "google_container_node_pool" "with_upgrade_settings":
          13: resource "google_container_node_pool" "with_upgrade_settings" {
```
Any reason why this error occurs?